### PR TITLE
Feature/tree

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: NeverGreenGrove/NeverGreenGrove/shaders/fragment.shader [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
 void main()
 {
-   color = (object_color == 0) ? vec4(1.0f, 0.0f, 0.0f, 1.0f) : // red
-           (object_color == 1) ? vec4(0.0f, 1.0f, 0.0f, 1.0f) : // green
-           (object_color == 2) ? vec4(0.0f, 0.0f, 1.0f, 1.0f) : // blue
-           (object_color == 3) ? vec4(0.5f, 0.5f, 0.5f, 1.0f) : // grey
-           (object_color == 4) ? vec4(1.0f, 0.9333f, 0.0f, 1.0f) : // pacman yellow
-           (object_color == 5) ? vec4(0.0f, 0.5f, 0.5f, 1.0f) : // teal
-           vec4(1.0f, 1.0f, 1.0f, 1.0f);
+   color = vertex_color;
```
Comment (prince-chrismc) [inline NeverGreenGrove/NeverGreenGrove/shaders/fragment.shader:9]: The new fragment shader will require a color VBO

**Comment by prince-chrismc** [comment]
I am adding the tree to the main function but Im not sure its worth it, any thoughts?

**Review by DanielPri — APPROVED** [review]
Looks good! color VBO will be necessary anyway, and if we dont have color we can just put something simple as a placeholder.

**Comment by prince-chrismc** [comment]
figured it will be color until we figure out texturing

**Pull request merged**